python/rainbow/simulators/prox_rigid_bodies/mass.py

```diff
@@ -295,6 +292,11 @@ def __rotate_to_model_space_orientation(prop):
     )
 
     d, R = np.linalg.eig(A)
+    if np.linalg.det(R) < 0:
```

Comment:
Should this code not be replacing the code in line 314?

The code we have starting from line 301 tries to make sure that the first index is always corresponding to the largest eigenvalue. 

The correction done in lines 313-314 is flawed, one should swap the indices as done in lines 296-299.